demisto_sdk/commands/common/hook_validations/description.py

```diff
@@ -147,28 +149,43 @@ def verify_demisto_in_description_content(self):
         Return:
             True if 'Demisto' does not exist in the description content, and False if it does.
         """
-
-        data_dictionary = get_yaml(self.file_path)
-        is_unified_integration = data_dictionary.get('script', {}).get('script', '') not in {'-', ''}
+        description_path = ''
         yml_line_num = 0
+        yml_or_file = ''
 
-        if is_unified_integration:
-            description_content = data_dictionary.get('detaileddescription', '')
+        # case 1 the file path is for an integration
+        if find_type(self.file_path) == FileType.INTEGRATION:
+            integration_path = self.file_path
+            data_dictionary = get_yaml(self.file_path)
+            is_unified_integration = data_dictionary.get('script', {}).get('script', '') not in {'-', ''}
 
-            with open(self.file_path, 'r') as f:
+            if is_unified_integration:
+                description_content = data_dictionary.get('detaileddescription', '')
+                yml_or_file = 'in the yml file'
 
-                for line_n, line in enumerate(f.read().split('\n')):
-                    if 'detaileddescription:' in line:
-                        yml_line_num = line_n + 1
-        else:
-            try:
-                description_path = glob.glob(os.path.join(os.path.dirname(self.file_path), '*_description.md'))[0]
+                # find in which line the description begins in the yml
+                with open(self.file_path, 'r') as f:
+                    for line_n, line in enumerate(f.read().split('\n')):
+                        if 'detaileddescription:' in line:
+                            yml_line_num = line_n + 1
 
-            except IndexError:
-                error_message, error_code = Errors.no_description_file_warning()
-                self.handle_error(error_message, error_code, file_path=self.file_path, warning=True)
-                return True
+            # if not found try and look for the description file path
+            else:
+                try:
+                    yml_or_file = 'in the description file'
+                    description_path = glob.glob(os.path.join(os.path.dirname(self.file_path), '*_description.md'))[0]
```

Comment:
no reason to use glob
```suggestion
                    description_path = self.file_path.replace('.yml', '_description.md')
```
or
```suggestion
                    description_path = f'{os.path.splitext(self.file_path)}_description.md'
```
then you just need to make sure the file exists using `os.path.isfile` instead of the try-except.